search/lib/convert/granules.js

```diff
@@ -77,41 +77,78 @@ function cmrGranToFeatureGeoJSON (event, cmrGran) {
     cmrGran.links.filter(l => l.rel === DOC_REL && !l.inherited && l.href.includes('opendap'))
   );
 
-  const linkToAsset = (l) => ({
-    name: l.title,
-    href: l.href,
-    type: l.type
-  });
+  const linkToAsset = (l) => {
+    if (l.title === undefined) {
+      return {
+        href: l.href,
+        type: l.type
+      };
+    } else {
+      return {
+        name: l.title,
+        href: l.href,
+        type: l.type
+      };
+    }
+  };
 
   const assets = {};
   if (dataLink) {
     assets.data = linkToAsset(dataLink);
   }
   if (browseLink) {
     assets.browse = linkToAsset(browseLink);
+
+    if (browseLink.href.includes('.png')) {
+      assets.browse.type = 'images/png';
```

Comment:
@DrewE84 , I just wanted to check with you that this was not a mime type but a path component.

Reply:
@fgabelmannjr This is a mime type, but it's addressed by a PR on the nasa repo that's waiting for final approval. I didn't want to change anything to prevent future merge conflicts when we bring nasa master into this repo after the PRs get merged.

Reply:
@DrewE84 , ahh this makes perfect sense then.  Do you have any sense for when that final PR will get merged?

Reply:
@fgabelmannjr I think we're just waiting on final review, then I can merge it